frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -45,11 +45,21 @@ export function OperatorValueSelect({
     operatorSelectProps,
     allowQueryingEventsByDateTime,
 }: OperatorValueSelectProps): JSX.Element {
-    const [currentOperator, setCurrentOperator] = useState(operator)
     const { propertyDefinitions } = useValues(propertyDefinitionsModel)
 
     const propertyDefinition = propertyDefinitions.find((pd) => pd.name === propkey)
 
+    // DateTime properties should not default to Exact
+    const startingOperator =
+        allowQueryingEventsByDateTime &&
+        propertyDefinition?.property_type == 'DateTime' &&
```

Comment:
Hmm, I would have thought `tsc` would flag this:

```suggestion
        propertyDefinition?.property_type == PropertyType.DateTime &&
```